canonical-websites/www.ubuntu.com
# Added redirects for iot verticals

## Done

* Added redirects for iot verticals

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    - [robotics](http://0.0.0.0:8001/robotics)
    - [gateways](http://0.0.0.0:8001/gateways)
    - [digital signage](http://0.0.0.0:8001/digital-signage)
- see that they go to the right page

## Issue / Card

Fixes #2532
